server/fileloader.mjs

```diff
@@ -39,16 +46,51 @@ async function downloadLink(link) {
   fs.writeFileSync(filename, JSON.stringify(linkStatus));
 }
 
+async function transformAssetsPathToRelative(obj) {
```

Comment:
Stuff like this belongs into fileupdater.mjs. And it might be easier to just do a replace on the JSON but it's fine like this. Would be nice to get rid of the duplicate code though.

Reply:
Sure! I cleaned up that function and moved it into `fileupdater.mjs`. Is [this](https://github.com/ArnoldSmith86/virtualtabletop/pull/731/commits/41b1b92372ee70e71733594c5225b18041185199) what you meant?

I chose to traverse the JSON object than modifying its raw string, to ensures a well-formed JSON object in the end.